crouton.php

```diff
@@ -945,23 +955,24 @@ function admin_options_page() {
 					<div style="padding: 0 15px;" class="postbox">
 						<h3>Include Extra Meetings</h3>
 						<div class="inside">
-							<?php if ($this_connected) { ?>
-								<?php $extra_meetings_array = $this->get_all_meetings($this->options['root_server']); ?>
-							<?php } ?>
 							<p class="ctrl_key" style="display:none; color: #00AD00;">Hold CTRL Key down to select multiple meetings.</p>
-							<select class="chosen-select" style="width: 100%;" data-placeholder="Select Extra Meetings" id="extra_meetings" name="extra_meetings[]" multiple="multiple">
-								<?php if ($this_connected) { ?>
-									<?php foreach($extra_meetings_array as $extra_meeting){ ?>
-										<?php $extra_meeting_x = explode('|||',$extra_meeting); ?>
-										<?php $extra_meeting_id = $extra_meeting_x[3]; ?>
-										<?php $extra_meeting_display = substr($extra_meeting_x[0], 0, 30) . ' ' . $extra_meeting_x[1] . ' ' . $extra_meeting_x[2] . $extra_meeting_id; ?>
+							<select class="chosen-select" style="width: 100%;" data-placeholder="<?php echo ($this->options['extra_meetings_enabled'] == 0 ? 'Not Enabled' : 'Select Extra Meetings') ?>" id="extra_meetings" name="extra_meetings[]" multiple="multiple">
+								<?php if ($this_connected && $this->options['extra_meetings_enabled'] == 1) {
+										$extra_meetings_array = $this->get_all_meetings($this->options['root_server']);
+										foreach($extra_meetings_array as $extra_meeting){
+											$extra_meeting_x = explode('|||',$extra_meeting);
+											$extra_meeting_id = $extra_meeting_x[3];
+											$extra_meeting_display = substr($extra_meeting_x[0], 0, 30) . ' ' . $extra_meeting_x[1] . ' ' . $extra_meeting_x[2] . $extra_meeting_id; ?>
 										<option <?php echo ($this->options['extra_meetings'] != '' && in_array($extra_meeting_id, $this->options['extra_meetings']) ? 'selected="selected"' : '') ?> value="<?php echo $extra_meeting_id ?>"><?php echo esc_html($extra_meeting_display) ?></option>
 									<?php } ?>
-								<?php } else { ?>
+								<?php } elseif (!$this_connected ) { ?>
 									<option selected="selected" value="none"><?php echo 'Not Connected - Can not get Extra Meetings'; ?></option>
 								<?php } ?>
```

Comment:
there is another elseif condition to show that it's not enabled

Reply:
yes there i sa problem with that, I went a different route. one of the things with doing that is that by adding an option that says Not Enabled it will essentially be set, so next time someone hits save the checkbox will be enabled weather they want it to or not.

Reply:
Not seeing the same.

Reply:
how is this any different than "Not Connected" being saved?

Reply:
that would create an issue, but if oyur not connected you have other bigger issues and data wouldnt be pulled anyway

Reply:
ok i see, i'll update bread to follow suit

Reply:
we didnt have the checkbox before, the checkbox logic says that if an option item exists enable. so by putting an option that says Not Enabled and hit save your saying reenable.

Reply:
yea I was just about to post a comment on that bread commit when you replied here